core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -543,6 +544,7 @@ private int registerGroupId(HashGenerator hashGenerator, Page page, int position
 
     private static final class DictionaryLookBack
     {
+        private static final int INSTANCE_SIZE = toIntExact(ClassLayout.parseClass(DictionaryLookBack.class).instanceSize());
```

Comment:
`instanceSize` already returns int, no need to wrap with `toIntExact`

Reply:
The version of JOL(0.16) used in project `instanceSize` returns `long`.